action.yml

```diff
@@ -8,9 +8,16 @@ inputs:
     description: 'Additional metadata files that should be copied to the root of the deployment package. File patterns are supported, one pattern per line.'
     required: false
     default: ''
+  perform-nuget-verify:
```

Comment:
Not sure it makes sense to expose this setting. When would we ever be okay with this being skipped and letting a release go through?

Reply:
We wouldn't.  It was more of a defensive measure to help with rolling out the new version if we needed it.  It can be removed.

Reply:
Is it possible we would ever have a release that didn't have NuGet packages?

Reply:
I don't think so, but this action should just be able to silently skip past there being no NuGet packages, and not be something that we have to configure.

What does `dotnet nuget verify nugets/` do if `nugets` is empty? What about if it doesn't exist?

Reply:
It errors out.  I change the call to use the Get-ChildItem with ignore errors. This will deal with empty folders and non-existent folders better.